packages/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.4.4 (2022-04-06)
```

Comment:
Should this changelog have a aggregated changelogs cross plugins? like under 0.4.0

Reply:
Done.